options.go

```diff
@@ -82,8 +82,8 @@ type TransportOptions struct {
 
 // BenchmarkOptions are benchmark-specific options
 type BenchmarkOptions struct {
-	MaxRequests int           `short:"n" long:"max-requests" default:"1000000" description:"The maximum number of requests to make"`
-	MaxDuration time.Duration `short:"d" long:"max-duration" default:"0s" description:"The maximum amount of time to run the benchmark for"`
+	MaxRequests int           `short:"n" long:"max-requests" default:"0" description:"The maximum number of requests to make. Values <= 0 imply no limit."`
```

Comment:
This is not right since the unlimited flag is only set if max-requests is zero, but the comment here implies that -1 would work.